docs/development/tools/key-management.md

```diff
@@ -0,0 +1,48 @@
+---
+id: "key-management"
+title: "Key Management"
+---
+
+Keys in Substrate are stored in the keystore in the file system. To store keys into this keystore,
+you need to use one of the two provided RPC calls. If your keys are encrypted or should be encrypted
+by the keystore, you need to provide the key using one of the cli arguments `--password`, `--password-interactive` or `--password-filename`.
+
+## Auto-generated Keys
+
+For most users who want to run a validator node, the `author_rotateKeys` RPC call is sufficient.
+The RPC call will generate `N` Session keys for you and return their public keys. `N` is the number
+of session keys configured in the runtime. The output of the RPC call can be used as input for the
+ `session::set_keys` transaction.
+
+```bash
+$ curl -H 'Content-Type: application/json' --data '{ "jsonrpc":"2.0", "method":"author_rotateKeys", "id":1 }' localhost:9933
+```
+
+## Fixed Keys
+
+If the session keys need to match a fixed seed, they can be set individually key by key. You can use
+[subkey Tool](development/tools/subkey.md#generating-keys) to generate the needed seed and public key
+that can be passed into the RPC call with a key type.
+
+```bash
+$ subkey generate
```

Comment:
This is not a guide to using Subkey to generate keys. Indeed, you have to generate the correct type for each Session key. You cannot just run `subkey generate` and pass that in and expect it to work.